src/Entity/Product.php

```diff
@@ -95,12 +96,12 @@ public function setDateOfLastModification(\DateTimeInterface $dateOfLastModifica
         return $this;
     }
 
-    public function getProductCategory(): ?productCategory
+    public function getProductCategory(): ?ProductCategory
```

Comment:
To nie powinno być zmienione - ten zapis oznacza jaka klas zostanie zwrócona. Nie istnieje klasa producktCategory, ale ProductCategory już tak